massa-node/base_config/config.toml

```diff
@@ -200,6 +204,8 @@
     ip_list_max_size = 10000
     # refuse consecutive bootstrap attempts from a given IP when the interval between them is lower than per_ip_min_interval milliseconds
     per_ip_min_interval = 180000
+    # Read-Write limitation for a connection in bytes per seconds (about the bootstrap specifically)
+    max_bit_read_write = 5_000_000
```

Comment:
I think we should either have read/write separate here as well, or remove the separation in the network binders as well

Let's put this to 20_000_000 for now